crates/ruff_linter/src/rules/pydocstyle/rules/backslashes.rs

```diff
@@ -67,11 +76,15 @@ pub(crate) fn backslashes(checker: &mut Checker, docstring: &Docstring) {
     if memchr_iter(b'\\', bytes).any(|position| {
         let escaped_char = bytes.get(position.saturating_add(1));
         // Allow continuations (backslashes followed by newlines) and Unicode escapes.
-        !matches!(escaped_char, Some(b'\r' | b'\n' | b'u' | b'N'))
+        !matches!(escaped_char, Some(b'\r' | b'\n' | b'u' | b'U' | b'N'))
     }) {
-        checker.diagnostics.push(Diagnostic::new(
-            EscapeSequenceInDocstring,
+        let mut diagnostic = Diagnostic::new(EscapeSequenceInDocstring, docstring.range());
+
+        diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
```

Comment:
This does change the meaning (content) of the docstring, so I think it should at least be an `unsafe_edit`, right?

Reply:
Good point! I'll modify accordingly.